<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModuleMasterPage</class>
 <widget class="QWizardPage" name="ModuleMasterPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Select Master/Slave </string>
  </property>
  <property name="subTitle">
   <string>The module may be selected as master or slave. In the system is only one master allowed. The slaves may be terminated but one of the slaves must be terminated.</string>
  </property>
  <widget class="QCheckBox" name="masterCheckBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>73</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Master</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="terminateCheckBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>73</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Terminate</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="externsyncCheckBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>External Synchronisation</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>masterCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>terminateCheckBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>masterCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>externsyncCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
